updater.go

```diff
@@ -89,9 +90,12 @@ func (u *Updater) update(ctx Context, options UpdateOptions) (*Update, *Error) {
 		return update, promptErrPtr(err)
 	}
 	switch updateAction {
-	case UpdateActionApply, UpdateActionAuto:
-		// Continue
+	case UpdateActionApply:
+		ctx.ReportAction(UpdateActionApply, options)
+	case UpdateActionAuto:
+		ctx.ReportAction(UpdateActionAuto, options)
```

Comment:
is it important to distinguish between... this update was applied without user being prompted... in which case my reporting is correct... if the goal is to report the "auto" setting then we need to change this request to not mix an action with state?

So params would be?
```
action: "apply", "snooze", "cancel", "auto"
auto_apply: 1/0
```

If apply is selected and user doesn't enable auto: `action=apply&auto_apply=0`
If apply is selected and user turns on auto apply: `action=apply&auto_apply=1`
If there was no prompt, user turns on auto apply: `action=auto&auto_apply=1`

If we ever see auto with auto_apply=0 then that would indicate a problem.